src/include/ceph_features.h

```diff
@@ -110,7 +110,7 @@ DEFINE_CEPH_FEATURE(18, 1, CRUSH_TUNABLES)   // 3.6
 DEFINE_CEPH_FEATURE_RETIRED(19, 1, CHUNKY_SCRUB, JEWEL, LUMINOUS)
 DEFINE_CEPH_FEATURE(19, 2, OSD_PGLOG_HARDLIMIT)
 DEFINE_CEPH_FEATURE_RETIRED(20, 1, MON_NULLROUTE, JEWEL, LUMINOUS)
-
+DEFINE_CEPH_FEATURE(20, 2, OSD_COPY_FILE) // 5.2
```

Comment:
is this feature bit still needed?

Reply:
Yes, we still need a way for the (kernel) client to figure out if it's safe to use the COPY_FROM op in copy_file_range().  A feature bit seems to be the obvious thing to use.

Reply:
do you intend to backport this fix?  if so, how far?  the feature bit needs to be chosen very carefully... this bit, for instance, is really only usable post-luminous.

if no backport is needed, then we can just reuse the SERVER_OCTOPUS bit to signal this support.

Reply:
@liewegas Thank you for your review.

TBH I don't really care about backporting this fix, thus reusing the SERVER_OCTOPUS bit is good enough.  So, IIUC you're suggesting to completely drop the definition of this OSD_COPY_FILE feature and simply test for Octopus, right?  Or would you rather keep the COPY_FILE feature but overlapping with the SERVER_OCTOPUS?

Reply:
Let's just re-use SERVER_OCTOPUS--simpler.

Reply:
I've just pushed my branch that include this change.  Thanks for your review @liewegas